test/code_size/hello_world_wasm2js.json

```diff
@@ -1,8 +1,8 @@
 {
   "a.html": 323,
   "a.html.gz": 253,
-  "a.js": 1060,
-  "a.js.gz": 636,
-  "total": 1383,
-  "total_gz": 889
+  "a.js": 1264,
```

Comment:
Pretty large increase..?

Reply:
Yeah, but its a fixed overhead.  From the diff it looks like closure is no longer minifying the WebAssmebly.xxx stuff.     I'm not sure we care about this do we?

Reply:
It's a fixed amount but it still seems worrying. Is it worth the benefit of removing the warning?